mmedit/datasets/pipelines/sr_resize.py

```diff
@@ -7,6 +7,76 @@
 from ..registry import PIPELINES
 
 
+@PIPELINES.register_module()
+class SRResize:
+    """Resize image, include up sampling and down sampling.
+
+        The result will be saved in the same key (output_key == input_key)
```

Comment:
Image will be loaded from the input_key and the result will be saved in the specified output_key (can equal to input_key).